src/sync/probspecs.nim

```diff
@@ -10,6 +10,7 @@ type
 
   ProbSpecsTestCase* = object
     json*: JsonNode
+    parents*: seq[JsonNode]
```

Comment:
We need to keep track of the parents to iterate over their descriptions later on.